### PR TITLE
Setting up PHP Package Publishing

### PR DESCRIPTION
**Pull request by EvanLovely**
This will push to read-only repos when we test. Currently set up is:

- `bolt/core-php` published to https://github.com/bolt-design-system/core-php

### CODE REVIEW

**Comment by EvanLovely** [comment]
@sghoweri  OK, if I did this right, then the change to the readme here e394c05 should not get pushed to the read-only repo until after it is merged into master.

File: scripts/update-read-only-git-repos.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Must be ran from `../` ie `./scripts/update-read-only-git-repos.sh`
+if [[ $TRAVIS == 'true' ]]; then
+  if [[ $TRAVIS_PULL_REQUEST == 'false' ]]; then
+    branch_name=$TRAVIS_BRANCH
+  else
+    branch_name=$TRAVIS_PULL_REQUEST_BRANCH
+  fi
+
+  # @todo Update to only trigger on tags
```
Comment (sghoweri) [inline scripts/update-read-only-git-repos.sh]: Next pass at this?

File: packages/core-php/readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Bolt PHP Core
```
Comment (sghoweri) [inline packages/core-php/readme.md]: @todo: add more info on what this subtree split of the main repo handles

**Comment by sghoweri** [comment]
> @sghoweri OK, if I did this right, then the change to the readme here e394c05 should not get pushed to the read-only repo until after it is merged into master.

@EvanLovely well then I think a little experiment is in order 😉

**Comment by sghoweri** [comment]
@EvanLovely when was this script ^ supposed to run? Or maybe the better question is what was supposed to trigger it?

 If the answer is Travis, don't we need to add this script to our travis config?

**Pull request merged**